docs/articles/guides/custom-icons.md

```diff
@@ -0,0 +1,107 @@
+# Register Icon pack
```

Comment:
Looks like you started branch from other PRs branch. It's file from #1807.
Please remove.